src/main/java/org/ihiw/management/service/dto/ProjectDTO.java

```diff
@@ -78,7 +85,7 @@ public ProjectDTO(Project project) {
         this.modifiedBy = project.getModifiedBy();
         this.labs = project.getLabs();
         this.leaders = project.getLeaders();
-
+        this.uploads = project.getUploads();
```

Comment:
For the actual issue, this does not change anything. You may either want to remove it, or keep it, if you plan to make use of it in the future (e.g. showing all uploads in the project view).